assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py

```diff
@@ -817,6 +909,13 @@ def finetune(args: Namespace):
     args.save_strategy = args.evaluation_strategy
     args.save_steps = args.eval_steps
 
+    if args.task_name in [Tasks.NLP_MULTICLASS, Tasks.NLP_MULTILABEL, Tasks.NLP_NER]:
```

Comment:
Change this to a function _def setup_automl_nlp(args) -> None_

Reply:
added